src/lib/component/partial/ExpandableKeyValueChip/ExpandableKeyValueChip.js

```diff
@@ -0,0 +1,90 @@
+import React from "/vendor/react";
+import PropTypes from "prop-types";
+import {
+  Dialog,
+  Button,
+  DialogTitle,
+  DialogContent,
+  DialogActions,
+  Slide,
+  withStyles,
+} from "/vendor/@material-ui/core";
+import { OpenInNewIcon } from "/lib/component/icon";
+import { ModalController } from "/lib/component/controller";
+import { CodeBlock, CodeHighlight } from "/lib/component/base";
+import { KeyValueChip } from "/lib/component/partial";
```

Comment:
I think this is causing the circular dep error when building.

Reply:
Ah, thanks for finding that.